package.json

```diff
@@ -22,7 +22,7 @@
         "run-graphql-demo-watch": "concurrently \"yarn run watch\" \"yarn run run-graphql-demo\"",
         "run-crm": "cd examples/crm && yarn start",
         "build-crm": "cd examples/crm && yarn build",
-        "storybook": "start-storybook -p 9009",
+        "storybook": "start-storybook -p 9010",
```

Comment:
why?

Reply:
Had a known issue with storybook only returning 404. Changing the port solved it